Library/Homebrew/dev-cmd/typecheck.rb

```diff
@@ -47,6 +50,22 @@ def typecheck
         return
       end
 
+      lines_to_keep = []
```

Comment:
`if args.prune_files_list?` ?

Reply:
> Yeh, let's just make it default I reckon. I think it's fine if it gets "lost".

Mike said to just make it a default thing when `brew typecheck` runs.

Reply:
Instead of having users to commit these changes, how about we have a separate `prune_files_list` flag that we can include in the tapioca cron job?

Reply:
That's a good idea, then there's no human involvement! I did this in a106fbc.